python/cudf/cudf/io/parquet.py

```diff
@@ -561,6 +565,118 @@ def read_parquet(
         **kwargs,
     )
 
+    # Can re-set the index before returning if we filter
+    # out rows from a DataFrame with a default RangeIndex
+    # (to reduce memory usage)
+    post_filters = post_filters or filters
```

Comment:
So if there are no `post_filters` we apply `filters` twice (because the parquet reader also does?) Otherwise only once?

Reply:
`filters` would have already been used to drop entire row-groups, but it would not have been used for row-wise filtering.  Therefore, we assume that users will want the same filters to be applied post-IO to make sure that any previously-missed rows are filtered out.  In contrast, when dask_cudf is calling this function, they will **only** want the post-IO filtering, because dask already drops unnecessary row-groups at graph-construction time.